### PR TITLE
improve condWait documentation

### PR DESCRIPTION
**Pull request by xvuko**

### CODE REVIEW

File: kernel/syscalls/sync.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ GETFROMSTACK(ustack, unsigned int, m, 1);
 GETFROMSTACK(ustack, time_t, timeout, 2);
 ````
 
-Waits on conditional given by 'h' for number of microseconds giveb by `timeout`. Before suspending a calling thread execution mutex identified by `m` handle is unlocked to enable other thread modifying variables used to check condtionals after conditional signalisation. When conditional variable is signaled mutex `m` is locked.
+Waits on conditional given by 'h' for number of microseconds giveb by `timeout`. Before suspending a calling thread execution mutex identified by `m` handle is unlocked to enable other thread modifying variables used to check condtonals after conditional signalisation. When conditional variable is signaled mutex `m` is locked.
```
Comment (gerard5) [inline kernel/syscalls/sync.md:51]: giveb -> given

**Review by agkaminski — APPROVED** [review]

**Review by niewim19 — COMMENTED** [review]
I`am not sure about `Arguments` section addition. Only two function docs use it (`bind` and `closelog` for some reason) and I think they should be corrected not to use it. It is just a matter of coherence between function docs. This section might be handy - its just we don`t use it :)

**Comment by xvuko** [comment]
> This section might be handy - its just we don`t use it :)
We should.  Searching description text each time arguments need to be checked is not a good option.

**Comment by niewim19** [comment]
> > This section might be handy - its just we don`t use it :)
> 
> We should. Searching description text each time arguments need to be checked is not a good option.

I agree that searching the whole description for arguments explanation is not effective and we may use the `Arguments` section if we decide on. 

You don't have to search through the whole description to get the grip on the arguments. The formula of current docs joins brief explanation of the function with arguments specification (_in italic_) in the first one or two sentences of the description. The `Arguments` section may be handy but it often doubles the information provided by the `Description` and declaration section.

Nevertheless, we try to keep the documentation in the _opengroup_ style which does not use `Arguments` list. This will make this function stand out which is anti-standarizational.

**Review by anglov — COMMENTED** [review]
In general – I like arguments section and I think it could be beneficial to have it,  but indeed it's not exists in documentation of other *nix systems  (linux, *BSD, SunOs)  and it's not common to our documentation.

**Comment by xvuko** [comment]
> You don't have to search through the whole description to get the grip on the arguments. The formula of current docs joins brief explanation of the function with arguments specification (_in italic_) in the first one or two sentences of the description. The `Arguments` section may be handy but it often doubles the information provided by the `Description` and declaration section.

Ok, this makes sense. I will change commit to use this style.

**Pull request merged**